src/components/ContactCard/ContactForm.jsx

```diff
@@ -1,5 +1,106 @@
 import React from "react";
+import PropTypes from "prop-types";
+import { Form, Field } from "react-final-form";
+import Icon from "cozy-ui/react/Icon";
+import { Button } from "cozy-ui/react/Button";
+import IconEmail from "../../assets/icons/email.svg";
+import IconPhone from "../../assets/icons/phone-number.svg";
+import IconAddress from "../../assets/icons/location.svg";
+import IconCozy from "../../assets/icons/cozy.svg";
+import IconCompany from "../../assets/icons/company.svg";
+import IconBirthday from "../../assets/icons/calendar.svg";
+import IconNotes from "../../assets/icons/comment.svg";
 
-const ContactForm = () => <div>Contact Form </div>;
+const fields = [
```

Comment:
We need an array to hold contact creation form fields. Be free to add what are necessary to customize fields. Until now I spotted `icon`, a key that will be translated and an input type. I guess there will be at least something to describe the multiple values capabilities (email, phone), and maybe some kind of validation functions.